docs/src/pages/[platform]/getting-started/migration/migration.vue.mdx

```diff
@@ -51,16 +51,16 @@ If you were using `I18n` to translate those keys, please update your translation
 
 ## 1.x to 2.x
 ## Installation
-Install the 2.x version of the `@aws-amplify/ui-vue` library.
+Install the 2.x version of the `@aws-amplify/ui-vue` library and the 5.x version of the `aws-amplify` library.
```

Comment:
Did you validate the peerDependency version of `aws-amplify` at ui-vue version 2?